packages/runtime/test/lib/testUtilsWithInactiveModules.ts

```diff
@@ -0,0 +1,136 @@
+import { IResponse, RelationshipCreationChangeRequestContent } from "@nmshd/content";
+import { CreateOutgoingRequestRequest, LocalRequestDTO, MessageDTO, RelationshipDTO } from "src";
+import { TestRuntimeServices } from "./RuntimeServiceProvider";
+import { exchangeMessageWithRequest, exchangeTemplate, syncUntilHasMessageWithResponse } from "./testUtils";
+
+export interface LocalRequestWithSource {
+    request: LocalRequestDTO;
+    source: string;
+}
+
+export interface ResponseMessagesForSenderAndRecipient {
+    rResponseMessage: MessageDTO;
+    sResponseMessage: MessageDTO;
+}
+
+export interface RelationshipRequestWithRelationshipIfAccepted {
+    request: LocalRequestDTO;
+    relationship: RelationshipDTO | undefined;
+}
+
+export async function exchangeMessageAndReceiverRequiresManualDecision(
```

Comment:
Confusing name :( My brain doesn't know how to parse it. Can you come up with something else?

Reply:
I'm now at "exchangeMessageWithRequestAndRequireManualDecision" - however this is only used in the more technical no-module case. That outsiders don't understand it seems more of a feature than a bug

Reply:
Sounds good.